packages/webpack/src/loader.ts

```diff
@@ -6,25 +6,21 @@ import {
   TranslationMessagesByKey,
 } from '@vocab/types';
 import { getDevLanguageFileFromTsFile, loadTranslation } from '@vocab/core';
-import { getOptions } from 'loader-utils';
+import { getOptions, stringifyRequest } from 'loader-utils';
+import type { LoaderContext as WebpackLoaderContext } from 'webpack';
```

Comment:
This package doesn't actually have any dependency on `webpack`, so you'll need to add one. I think the monorepo deps on `webpack` and `babel-loader` can be removed.

Reply:
This package should have a peer dependency on Webpack, because it's a Webpack loader. I don't think the loader works with Webpack 3 or 4.

I could enhance the locally defined `LoaderContext` but that is compatible only with certain versions of Webpack.

Reply:
Yeah it should have a peer dep on webpack.